integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/preaaggregate/PreAggregateUtil.scala

```diff
@@ -489,4 +489,44 @@ object PreAggregateUtil {
     }
     updatedPlan
   }
+
```

Comment:
This code is specific for inserting the data to aggregate table better to move this code in rules class

Reply:
moved the method to PreAggregationRules